modules/dropdown/js/dropdown_directive.js

```diff
@@ -394,15 +394,10 @@
             lxDropdown.isOpen = true;
 
             LxDepthService.register();
-
-            if (!lxDropdown.hover) {
-                scrollMask
-                    .css('z-index', LxDepthService.getDepth())
-                    .appendTo('body');
-                scrollMask.on('wheel', function preventDefault(e) {
-                    e.preventDefault();
-                });
-            }
+            
+            scrollMask.css('z-index', LxDepthService.getDepth()).appendTo('body');
+            
+            angular.element(scrollMask).on('click wheel touchmove ontouchstart', closeDropdownMenu);
```

Comment:
scrollMask is already an angular.element I think so no need to re-wrap it with jq?